third_party/gif_decoder/src/main/java/com/bumptech/glide/gifdecoder/GifDecoder.java

```diff
@@ -195,7 +195,7 @@ public void advance() {
      */
     public int getDelay(int n) {
         int delay = -1;
-        if ((n >= 0) && (n < header.frameCount)) {
+        if (0 <= n && n < header.frameCount) {
```

Comment:
Yeah I'd prefer 0 <= n < frameCount if the language supported it, but as is, I'm used to reading variable <=> number.